FitNesseRoot/DbFit/AcceptanceTests/JavaTests/TeradataTests/FlowMode/SimpleStoredProc/content.txt

```diff
@@ -1,6 +1,6 @@
 This stored proc tests output parameters - CalcLength returns string length as second output param
 
-|Execute Ddl|!-Replace Procedure CalcLength(in name varchar(100), out strlength numeric)
+|Execute Ddl|!-Replace Procedure CalcLength(in name varchar(100), out strlength bigint)
```

Comment:
As we're removing the `numeric` version here - do we have some other test which is still covering the `numeric` behavior?